solr/core/src/test/org/apache/solr/cloud/OverseerTest.java

```diff
@@ -127,7 +127,6 @@ public class OverseerTest extends SolrTestCaseJ4 {
   private final List<UpdateShardHandler> updateShardHandlers =
       Collections.synchronizedList(new ArrayList<>());
   private final List<CloudSolrClient> solrClients = Collections.synchronizedList(new ArrayList<>());
-  private static final String COLLECTION = SolrTestCaseJ4.DEFAULT_TEST_COLLECTION_NAME;
```

Comment:
Not sure this change is needed? It looks like this is doing the "right thing" already?

Reply:
Same argument as above...   If we learn, reading the tests, taht DEFAULT_TEST_COLLECTION_NAME means that the test using the default collection name, and that you don't need to think about it, then let's use that name everywhere...